core/services/workflows/syncer/fetcher.go

```diff
@@ -0,0 +1,59 @@
+package syncer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/validation"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/webapi"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	ghcapabilities "github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/capabilities"
+)
+
+func NewFetcherFunc(
```

Comment:
@agparadiso Does this need to be plugged in somewhere?

Reply:
IIUC this will be passed to the [NewWorkflowRegistry](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/workflows/syncer/workflow_registry.go#L144) method, but this is not used anywhere yet.